prompt2model/utils/dataset_utils.py

```diff
@@ -31,3 +33,50 @@ def get_dataset_size(dataset_name):
         if size_dict == {}
         else "{:.2f}".format(size_dict["dataset"]["num_bytes_memory"] / 1024 / 1024)
     )
+
+
+def make_combined_datasets(
+    dataset_list: list[datasets.Dataset], dataset_type: str = "inp_out"
+) -> datasets.Dataset:
+    """Comnine multiple datasets into one.
+
+    Args:
+        dataset_list: List of datasets to combine.
+        dataset_type: Type of dataset to combine. Can be "text" or "inp_out".
+        "text" is for combining datasets with a single column "text".
+        "inp_out" is for combining datasets with 2 columns "input_col" and "output_col".
```

Comment:
nit - rename the column to `input_out` unless there's a practical reason not to, since this seems like an unnecessary abbreviation